Mta 6233 (#5257)
scaleway/docs-content: pages/environmental-footprint/additional-content/block-storage-env-footprint.mdx

---

This page includes the specificities of the environmental footprint calculation for Block Storage.

## Calculation aspects

The total estimated impact of Block Storage volumes is calculated by integrating the:

- **Dedicated manufacturing impact** - Related to the physical servers hosting the storage volumes.
- **Related manufacturing impact** - The impact of the tools used to manage the Block Storage offers: control plane, monitoring, internal load balancers, and others.
- **Energy usage impact** - The average electricity consumption of storage servers and their energy efficiency and of the tools used to manage the Block Storage offers.

### Ratio per storage volume

To allocate the impact of a Block Storage volume, we apply the unit resulting from the RCP allocation rule to all calculation steps, the `bls_ratio`:

```
bls_ratio = VOLsto/VolstoPool
```

- **VOLsto** - represents the **reserved volume** per user. In Block Storage, your data is replicated three times on multiple nodes to guarantee availability, so the **VOLsto** is applied with factor of 3 to account for replication.
- **VolstoPool** - represents the total storage volume reserved for Block Storage.

> In the example below we consider a **Volsto** of 600 GB and a **VolstoPool** of 1000 GB.

Reviewer: ```suggestion
In the example below we consider a **VOLsto** of 600 GB and a **VolstoPool** of 1000 GB.
```

```
bls_ratio = 600/1000 = 0,6
```

### Manufacturing impact

The calculation of the manufacturing impact of Block Storage volumes is based on:

- The fabrication impact of servers that host user storage
- The fabrication impact of servers used by the Scaleway Block Storage team to manage the service - on both the Control and Data planes

The calculation is the same used for [measuring the impact of manufacturing for Bare Metal servers](/environmental-footprint/additional-content/environmental-footprint-calculator/#customer-servers), but for Block Storage we also apply the `bls_ratio` to break the impact down to only the allocated volumes.

#### Calculation example

> For the example below, we consider a single server with a lifespan of 6 years (52 560 hours) and power of 110w. We measure the impact of 100 hours and consider through [Boavizta estimates](https://www.boavizta.org/) that the impact of the manufacturing of this server is 850 kgCo2e. Also considering a `bls_ratio` of 0,6, we make the following calculation:

Reviewer 2: ```suggestion
For the example below, we consider a single server with a lifespan of 6 years (52 560 hours) and power of 110w. We measure the impact of 100 hours and consider through [Boavizta estimates](https://www.boavizta.org/) that the impact of the manufacturing of this server is 850 kgCo2e. Also considering a `bls_ratio` of 0.6, we make the following calculation:
```

---
meta:
  title: Block Storage footprint calculation
  description: This page explains how Scaleway calculates the environmental footprint of its Block Storage in detail.
content:
  h1: Block Storage footprint calculation
  paragraph: This page explains how Scaleway calculates the environmental footprint of its Block Storage in detail.
tags: block-storage environmental-footprint impact datacentre
dates:
  validation: 2025-07-10
  posted: 2025-07-10
categories:
  - environmental-footprint
  - block-storage
---

This page includes the specificities of the environmental footprint calculation for Block Storage.

## Calculation aspects

The total estimated impact of Block Storage volumes is calculated by integrating the:

> - **Manufacturing impact** - Related to the physical servers hosting the storage volumes.

Reviewer 3: ```suggestion
- **dedicated manufacturing impact** - Related to the physical servers hosting the storage volumes.
```

> - **Usage Impact** - The impact of the tools used to manage the Block Storage offers: control plane, monitoring, internal load balancers, and others.

Reviewer 3: ```suggestion
- **Related Manufacturing Impact** - The impact of the tools used to manage the Block Storage offers: control plane, monitoring, internal load balancers, and others.
```

> - **Energy usage impact** - The average electricity consumption of storage servers and their energy efficiency.

Reviewer 3: ```suggestion
- **Energy usage impact** - The average electricity consumption of storage servers and their energy efficiency and of the tools used to manage the Block Storage offers.
```

### Ratio per storage volume

To allocate the impact of a Block Storage volume, we apply the unit resulting from the RCP allocation rule to all calculation steps, the `bls_ratio`:

```
bls_ratio = VOLsto/VolstoPool
```

- **VOLsto** - represents the **reserved volume** per user. In Block Storage, your data is replicated three times on multiple nodes to guarantee availability, so the **VOLsto** is applied with factor of 3 to account for replication.

> - **VolstoPool** - represents all the storage for Block Storage volumes on the server that hosts them (around 60%).

Reviewer 3: ```suggestion
- **VolstoPool** - represents the total storage volume reserved for the block storage product.
```

> In the example below we consider a **Volsto** of 600 and a **VolstoPool** of 1000.

Reviewer 3: ```suggestion
In the example below we consider a **Volsto** of 600 Gb and a **VolstoPool** of 1000 Gb.
```

```
bls_ratio = 600/1000 = 0,6
```

### Manufacturing impact

The calculation of the manufacturing impact of Block Storage volumes is based on:

- The fabrication impact of servers that host user storage
- The fabrication impact of servers used by the Scaleway Block Storage team to manage the service - on both the Control and Data planes

The calculation is the same used for [measuring the impact of manufacturing for Bare Metal servers](/environmental-footprint/additional-content/environmental-footprint-calculator/#customer-servers), but for Block Storage we also apply the `bls_ratio` to break the impact down to only the allocated volumes.

#### Calculation example

For the example below, we consider a single server with a lifespan of 6 years (52 560 hours) and power of 110w. We measure the impact of 100 hours and consider through [Boavizta estimates](https://www.boavizta.org/) that the impact of the manufacturing of this server is 850 kgCo2e. Also considering a `bls_ratio` of 0,6, we make the following calculation:

```
VolumeManufacturingImpact = (100 / 52560) * 850 * 0,6 = 0,97 kgCO2e
```

> 

Reviewer 3: we can add a conclusion here : "so the dedicated manufacturing impact for a usage of 600Gb during 100 hours in this exemple is 0,97 kgCo2e

### Usage impact

The calculation of the usage impact of Block Storage volumes is based on:

> - The usage of servers that host user storage

Reviewer 3: ```suggestion
- Impact of electricity consumption of servers that host user storage
```

> - The usage of infrastructure and service management servers by the Scaleway Block Storage team to run the Block Storage service. In this case we consider both the Control and Data planes.

Reviewer 3: ```suggestion
- Impact of electricity consumption of infrastructure and service management servers by the Scaleway Block Storage team to run the Block Storage service. In this case we consider both the Control and Data planes.
```

> - The impact of the electrical power dedicated to Block Storage across the infrastructure

Reviewer 3: ```suggestion

```

> To calculate the usage, we consider the French energy mix and the datacenter's PUE.

Reviewer 3: ```suggestion
To calculate the impact of a kwh, we consider the French energy mix and the datacenter's PUE.
```

In the example below, they are O,065kgCo2e/kwh and 1,16, respectively. Considering an electrical consumption of 600wh, 100 hours of usage, and a `bls_ratio` of 0,6, the total usage impact comes down to:

```math
UsageImpact = (0,600 x 0,065) * 100 * 0,6 = 2,34 kgCo2e
```

> 

Reviewer 3: We can also add a conclusion here
